src/containerapp/azext_containerapp/_params.py

```diff
@@ -66,6 +66,10 @@ def load_arguments(self, _):
         c.argument('managed_certificates_only', options_list=['--managed-certificates-only', '-m'], help='List managed certificates only.')
         c.argument('private_key_certificates_only', options_list=['--private-key-certificates-only', '-p'], help='List private-key certificates only.')
 
+    with self.argument_context('containerapp env dapr-component init') as c:
```

Comment:
```suggestion
    with self.argument_context('containerapp env dapr-component init', is_preview=True) as c:
```

Reply:
![image](https://github.com/Azure/azure-cli-extensions/assets/22132944/5267b89e-3ec2-4f79-a41c-4b0c323f071a)
Do we need to mark the args individually as preview? The command itself is in preview.